index.js

```diff
@@ -303,32 +303,38 @@ Transformer.prototype.renderAsync = function (str, options, locals, cb) {
   }
 };
 Transformer.prototype.renderFile = function (filename, options, locals) {
-  if (typeof this._tr.renderFile === 'function') {
+  if (!this.can('renderFile')) { // *Async, *Client
+    throw new Error('The Transform "' + this.name + '" does not support rendering synchronously.');
```

Comment:
If it can't `.renderFile()`, then doesn't it implement the fallback below?

``` javascript
return tr.normalize(this._tr.render(tr.readFileSync(filename, 'utf8'), options, locals));
```